src/python_testing/global_attribute_ids.py

```diff
@@ -38,3 +39,95 @@ def to_name(self) -> str:
             return "FeatureMap"
         if self == GlobalAttributeIds.CLUSTER_REVISION_ID:
             return "ClusterRevision"
+
+
+@dataclass
+class IdRange:
```

Comment:
Could we use `range(start, stop)` as a python object instead of having a separate min/max class with contains?

Reply:
sure. I did it like basically so I could have the contains function. Are you thinking of making the range the datamember? or using something like STANDARD_PREFIX = range(0x0000, 0x0000+1) and having a prefix_contains and suffix_contains functions?

Reply:
I thought complete replacing it. If this was done to have 'includes the end' range, then maybe a comment saying that would help (to make it clear why the python range object did not directly fit the bar).

Reply:
added a comment.